### PR TITLE
Wiki Reorg: AnV Protocol

### PR DESCRIPTION
**Pull request by filippoweb3**
See [this issue](https://github.com/w3f/polkadot-wiki/issues/4376) and [this google doc](https://docs.google.com/document/d/1Fl8ReBIzcaQEOyF6s03d7MEcgk8aTVEt7xjmzAk_k5I/edit) for more information about the Wiki reorg.

### CODE REVIEW

**Comment by filippoweb3** [comment]
@DrW3RK @joepetrowski I've put together the information about the process of getting a parablock included in the Relay Chain. The page merges info that was previously scattered on the Validator Page and the AnV Page. I am seeking a first round of review. The page can be split into two later on, but I think it is a good thing to have everything in one place for now so that we can make the whole story more fluid and cohesive. Let me know what you think

File: docs/learn/learn-parachains-protocol.md [hunk 1]
```diff
@@ -156,22 +295,233 @@ para-blocks are shown as yellow squares that become grey in color.
...
+another fork containing good blocks. Dealing with a bad parablock includes the following stages:
+
+- Detection: the bad block must be detected by honest validators.
+- Escalation: the honest validators must send that block for checks to all validators. A
+  [dispute](#disputes) starts.
+- Consequences: the chain is reverted and all malicious validators are slashed.
+
+The result of the dispute must be transplantable to all other forks so that malicious validators are
+slashed in all possible histories and so that honest validators will ignore any forks containing
+that parablock.
```
Comment (joepetrowski) [inline docs/learn/learn-parachains-protocol.md]: Major red flags reading these paragraphs. What does it mean to be on the Relay Chain (also, be consistent with style) if not finalized? What is the difference between being ignored in favor of another fork vs. being reverted? Why would a block's validation influence sending that block to others? (I.e., the block should already be available, why would they need to send it? They can just send a dispute and others can request it to get involved.)

File: docs/learn/learn-parachains-protocol.md [hunk 2]
```diff
@@ -156,22 +295,233 @@ para-blocks are shown as yellow squares that become grey in color.
...
+
+:::info Parablocks vs Relay-Chain blocks
+
+It is important to understand that a relay chain block contains many parablocks. Thus, it makes more
+sense to think of relay-chain blocks as having been approvead instead of parablocks that have been
+approved. A relay-chain block containing a bad parablock must be reverted, while a relay-chain block
+containing only approved parablocks can be considered approved as long as its parent relay-chain
+block is also approved. Thus, the validity of a relay-chain block depends on the validity of its
+ancestry.
```
Comment (joepetrowski) [inline docs/learn/learn-parachains-protocol.md]: Relay Chain blocks contain para headers, not blocks.

**Comment by filippoweb3** [comment]
@DrW3RK let me know what you think. Feel free to request additional reviewers.

**Pull request closed without merging**